mods/common/languages/en.ftl

```diff
@@ -33,6 +33,7 @@ notification-changed-map = { $player } changed the map to { $map }
 notification-option-changed = { $player } changed { $name } to { $value }.
 notification-you-were-kicked = You have been kicked from the server.
 notification-kicked = { $admin } kicked { $player } from the server.
+notification-anonymous-kicked = { $player } was kicked from the server.
```

Comment:
Why is this called "anonymous" when there is a player argument?

Reply:
See the translation above. Though I do agree that it could use a better name

Reply:
updated the above translation